net-misc/curl/metadata.xml

```diff
@@ -9,6 +9,8 @@
 		<flag name="alt-svc">Enable alt-svc support</flag>
 		<flag name="brotli">Enable brotli compression support</flag>
 		<flag name="http2">Enabled HTTP/2.0 support</flag>
+		<flag name="nghttp3">Enabled HTTP/3.0 support using <pkg>net-misc/nghttp3</pkg> and <pkg>net-misc/ngtcp2</pkg></flag>
+		<flag name="quiche">Enabled HTTP/3.0 support using <pkg>net-misc/quiche</pkg></flag>
```

Comment:
All those "enabled xxx support" descriptions seem weird. Maybe they could all be converted to "enable".

Reply:
I've updated them, thanks!